netlify.toml

```diff
@@ -716,3 +716,8 @@
 [[redirects]]
     from = "/docs/integrate/client/js/index"
     to = "/docs/integrate/client/js"
+
```

Comment:
☝️  the above redirects are unrelated to this issue and will be fixed in another PR